src/west/commands.py

```diff
@@ -422,6 +422,9 @@ def inf(self, *args, colorize: bool = False, end: str = '\n'):
                          is undefined or true, and stdout is a terminal, then
                          the message is printed in green.
         '''
+        if Verbosity.INF > self.verbosity:
```

Comment:
For some reason I find the other way reads much faster :-) Closer to English?

```suggestion
        if self.verbosity < Verbosity.INF
```